tests/scripts/test-driver.sh

```diff
@@ -26,29 +26,16 @@ function Usage()
     echo ""
     echo "--artifactpath            - (default={test-driver.sh's path}/../../) it should contain tests/ and Product/"
     echo ""
-    echo "Following options are needed when you want to tests of specific types. If you don't pass any one, unittest and verification will be run"
-    echo "--frameworktest           - (default=off) run framework test"
-    echo "--verification            - (default=on) run verification"
-    echo "--frameworktest_list_file - filepath of model list for test"
-    echo ""
     echo "Following option is only needed when you want to test benchmark."
     echo "--benchmark_onert_op     - (default=off) run benchmark per operation on onert"
```

Comment:
Remain option is not working correctly, but I remains this because we don't have ported script yet.